tests/ttnn/unit_tests/operations/test_silu.py

```diff
@@ -223,3 +229,35 @@ def test_wh_silu_relu(
         shard_orientation,
         op,
     )
+
+
+@skip_for_wormhole_b0()
+@pytest.mark.parametrize(
+    "batch_size, input_channels, input_height, input_width, ncores, shard_grid, shard_strategy, shard_orientation",
+    ((1, 5120, 32, 1, 32, (1, 32), ttnn.ShardStrategy.WIDTH, ttnn.ShardOrientation.ROW_MAJOR),),
+)
+@pytest.mark.parametrize("op", ["silu", "relu"])
+def test_silu_llm(
```

Comment:
can this test code be commonized with the test code above in the same file `test_wh_silu_relu`, `test_gs_silu_relu`? This just adds more test permutations, right?

Reply:
Done.